backends/CLNRest.ts

```diff
@@ -0,0 +1,282 @@
+import stores from '../stores/Stores';
+import LND from './LND';
+import TransactionRequest from '../models/TransactionRequest';
+import OpenChannelRequest from '../models/OpenChannelRequest';
+import VersionUtils from '../utils/VersionUtils';
+import Base64Utils from '../utils/Base64Utils';
+import { Hash as sha256Hash } from 'fast-sha256';
+import BigNumber from 'bignumber.js';
+import {
+    getBalance,
+    getOffchainBalance,
+    listPeers
+} from './CoreLightningRequestHandler';
+
+export default class CLNRest extends LND {
```

Comment:
```suggestion
export default class CLNRest  {
```

There is no reason to extend LND, but we should re-export the method `getUrl` somewhere-else

Reply:
Some views are breaking when we not extend from lnd, also all the actual execution of the fetch calls happen in the LND class so its not just getUrl. Will leave it as is.

Reply:
>also all the actual execution of the fetch calls happen in the LND class so its not just getUrl. 

What do you mean? This approach will leave many calls unused, and you'll likely need to rewrite a lot of code as well when you support commando (or you are not supporting it?).

TL;DR: I disagree with using CLN code that relies on LND. Even though the current API design is different, the previous design wasn't bad because c-lightning-REST emulates LND. However, clnrest/commando does not.

Reply:
We are not supporting Commando in this PR. 
The only thing the CLN class relies on LND is the actual "fetch" function to call the APIs. everything else is separate. I tried removing the LND class extension but that was breaking a couple of UI components. I didn't find the time to investigate that.

Reply:
Nvm, i figured out the issue. CLN no longer extends LND now.